my-profile-webapp/src/main/webapp/partials/emergency.html

```diff
@@ -15,45 +14,58 @@
-                <select id="phone-type{{$index}}" class="form-control" required ng-model="contact.phoneNumbers[$index].type">
+            <div class="col-xs-5 no-padding phone-type">
+                <label for="phoneType{{$index}}" ng-if="$index === 0">Type*</label>
+                <label for="phoneType{{$index}}" ng-if="$index !== 0">Type</label>
+                <select id="phoneType{{$index}}" name="phoneType{{$index}}" class="form-control" required ng-model="contact.phoneNumbers[$index].type" ng-class="{ 'has-error' : emergencyForm.phoneType0.$invalid && emergencyForm.phoneType0.$touched }">
                     <option value="mobile" selected="selected">Mobile</option>
                     <option value="home">Home</option>
                     <option value="work">Work</option>
                     <option value="other">Other</option>
                 </select>
             </div>
+            <p class="form-help" ng-show="emergencyForm.phone0.$invalid && emergencyForm.phone0.$touched && $index === 0">Phone number is required</p>
+            <p class="form-help" ng-show="emergencyForm.phoneType0.$invalid && emergencyForm.phoneType0.$touched && $index === 0">Phone type is required</p>
         </div>
         <button class="btn btn-default" type="text" ng-click="contact.phoneNumbers.push('')">Add another phone number</button>
         <div class="form-group">
-            <label for="email">Email</label>
-            <input class="form-control" id='email' type='text' required ng-model="contact.emails[0].value">
+            <label for="email">Email*</label>
+            <input class="form-control" id='email' name="email" type='text' required ng-model="contact.emails[0].value" ng-class="{ 'has-error' : emergencyForm.email.$invalid && emergencyForm.email.$touched }">
+            <p class="form-help" ng-show="emergencyForm.email.$invalid && emergencyForm.email.$touched">Email is required</p>
         </div>
         <div class="form-group">
-            <label for="relationship">Relationship</label>
-            <input class="form-control" id='relationship' type='text' required ng-model="contact.relationship">
+            <label for="relationship">Relationship*</label>
+            <input class="form-control" id='relationship' name="relationship" type='text' required ng-model="contact.relationship" ng-class="{ 'has-error' : emergencyForm.relationship.$invalid && emergencyForm.relationship.$touched }">
+            <p class="form-help" ng-show="emergencyForm.relationship.$invalid && emergencyForm.relationship.$touched">Relationship is required</p>
         </div>
-        <div ng-repeat="phone in contact.languages | limitTo:2 track by $index" class="form-group">
+        <div class="form-group">
+            <label for="comments">Communication Considerations / Other Comments</label>
+            <textarea class="form-control" id='comments' type='text' ng-model="contact.comments" placeholder="i.e. Speaks Spanish"></textarea>
```

Comment:
Consider "e.g." rather than "i.e.", and then consider "For example: " rather than "e.g.". ( [Oatmeal](http://theoatmeal.com/comics/ie) ).